subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApiDistributionResolver.groovy

```diff
@@ -96,7 +96,16 @@ class ToolingApiDistributionResolver {
         Dependency dep = resolutionServices.dependencyHandler.create(dependency)
         Configuration config = resolutionServices.configurationContainer.detachedConfiguration(dep)
         config.resolutionStrategy.disableDependencyVerification()
-        return config.files
+        int retryCount = 3
+        List<Throwable> exceptions = []
+        while (retryCount-- > 0) {
```

Comment:
❔ Should we wait a little bit before retrying? Maybe even use an exponential backoff?